src/bot.py

```diff
@@ -35,8 +35,11 @@
     ]
 )
 
+mininterval = config['mininterval']
+maxinterval = config['maxinterval']
+hours_to_wait = randint(mininterval, maxinterval) * 24

```

Comment:
Maybe it would be a good idea to add all multiplications in a single line

The `cooldown` variable is used for `time.sleep()`, which takes an integer that counts the **seconds** that need to elapse until the functionality of the bot resumes normally. So, we have the days, and we want to convert that into seconds. If we did that in one line, the code would get much more clear.